### ISSUE TITLE
Application on Deformable DETR

### ISSUE THREAD

**Issue by jihwanp** [shenyi0220/CP-Cluster#13]
Hi thanks for providing such a wonderful work.

One thing I'm curious about is if this methodology works for NMS-free detector (e.g, Deformable DETR) as well.
Have you tried it?

**Comment by shenyi0220** [comment]
No, you can try it. My gut thought is that it may still improve for those lightweight models(such as onenet, original detr), as those small models still suffer from duplicate boxes more or less. While I feel it won't take effect for strong models(such as swin-dino-detr)
